client.go

```diff
@@ -24,8 +28,26 @@ var userAgent = fmt.Sprintf(
 	Version, runtime.Version(), runtime.GOOS, runtime.GOARCH,
 )
 
-func newClient() *Client {
+func newClient(use_proxy bool) *Client {
 	// Return default HTTP client for now. @todo replace with defined client
+	if use_proxy {
```

Comment:
Hmm is this more of a personal usage? I am not sure how needed a feature is since Epic can already track you through the user account you're using. What more would adding an ability to proxy requests through Tor give?